r/tests/testthat/test-dplyr-summarize.R

```diff
@@ -392,44 +398,38 @@ test_that("quantile()", {
     "quantile() currently returns an approximate quantile in Arrow",
     fixed = TRUE
   )
+  # silence warnings
+  rlang::local_options(rlib_warning_verbosity = "quiet")
 
   # without groups
-  expect_warning(
```

Comment:
These big diffs are just removing `expect_warning()` and dedenting everything accordingly